python/version_details/resources/resources.qrc

```diff
@@ -0,0 +1,26 @@
+<RCC>
+  <qresource prefix="version_details">
+    <file>right_arrow_large.png</file>
+    <file>right_arrow.png</file>
+    <file>right_arrow_white.png</file>
+    <file>panel_empty_background.png</file>
+    <file>dock.png</file>
+    <file>dock_hover.png</file>
+    <file>sort_down_on.png</file>
+    <file>sort_up_on.png</file>
+    <file>sort_down.png</file>
+    <file>sort_up.png</file>
```

Comment:
make sure to include source images for all icons etc. We normally push a graffle file, but anything works. :)